test/it/uk/gov/hmrc/component/CaseSpec.scala

```diff
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-package it.uk.gov.hmrc.component
+package uk.gov.hmrc.component
```

Comment:
You did the same change in the front-end project and the unit tests were running twice.
Could you make sure this time you do not introduce the same mistake?

Reply:
It was the build.sbt change that caused that, not this